app/services/bulk_api_import/fhir_importable.rb

```diff
@@ -14,11 +14,24 @@ def timestamps
     }
   end
 
-  def translate_id(id, ns_prefix: "")
-    Digest::UUID.uuid_v5(Digest::UUID::DNS_NAMESPACE + ns_prefix, id)
+  def translate_id(id, org_id:, ns_prefix: "")
+    Digest::UUID.uuid_v5(Digest::UUID::DNS_NAMESPACE + org_id + ns_prefix, id)
   end
 
-  def translate_facility_id(id)
-    FacilityBusinessIdentifier.find_by(identifier: id, identifier_type: :external_org_facility_id).facility.id
+  def translate_facility_id(id, org_id:)
+    FacilityBusinessIdentifier
+      .joins(facility: :facility_group)
+      .where(identifier_type: :external_org_facility_id,
+        facility_business_identifiers: {identifier: id},
+        facility_groups: {organization_id: org_id})
+      .take.facility.id
+  end
+
+  def translate_patient_id(id, org_id:)
+    translate_id(
+      id,
+      org_id: org_id,
+      ns_prefix: "patient_business_identifier"
+    )
   end
```

Comment:
Added a new helper, since this particular `ns_prefix` found a lot of reuse across the FHIR resources.